web/js/config.js

```diff
@@ -36,6 +45,8 @@ export default {
             {"label":"RERUM Geolocator", "icon":"http://geo.rerum.io/geolocate/images/earth.gif", "view":"http://geo.rerum.io/geolocate/annotate.html", "description" : "A water bear that knows where it is." }
         ]
     },
+
+    // What are these here for? Do the users know about these?
```

Comment:
List the interfaces and technologies that will be available that need to be rendered for someone to find in the playground.  
Like
`{"label":"Transcribe Interface", "icon":"http://t-pen.org/TPEN/images/logo.jpg", "view":"http://t-pen.org/TPEN/transcribe.html", "description" : "A line by line transcriber suited for Manuscripts." }`
or
`{"label":"Transkribus Tech", "icon":"https://readcoop.eu/transkribus/images/logo.jpg", "view":"https://readcoop.eu/transkribus/auto-transcribe.html", "description" : "comprehensive platform for the digitisation, AI-powered recognition, transcription and searching of historical documents." }`